projects/engine/src/engine/next-gen-engine.ts

```diff
@@ -126,12 +126,6 @@ export class Engine {
       this.manager.getProfile(caller),
     ])
 
-    // Check if plugin FROM can call METHOD of plugin TO
```

Comment:
that is removed?

Reply:
Yes, as discussed in Gitter, most of the calls won't require any permission at the method level. It's easier to maintain and more flexible to let plugin developers defined when to ask for permission with `canCall`.